draft-demarco-oauth-status-assertions.md

```diff
@@ -338,10 +338,10 @@ in the following table.
 | **aud** | It MUST be set with the Issuer Status Assertion endpoint URL as value that identify the intended audience. | {{RFC9126}}, {{RFC7519}} |
 | **exp** | UNIX Timestamp with the expiration time of the JWT. It MUST be superior to the value set for `iat` . | {{RFC9126}}, {{RFC7519}}, {{RFC7515}} |
 | **iat** | UNIX Timestamp with the time of JWT/CWT issuance. | {{RFC9126}}, {{RFC7519}} |
-| **jti** | Unique identifier for the JWT.  | {{RFC7519}} Section 4.1.7 |
-| **cti** | Unique identifier for the CWT.  | {{RFC7519}} Section 4.1.7 |
-| **credential_hash** | Hash value of the Digital Credential the Status Assertion is bound to. | this specification |
-| **credential_hash_alg** | The Algorithm used of hashing the Digital Credential to which the Status Assertion is bound. The value SHOULD be set to `sha-256`. | this specification |
+| **jti** | Unique identifier for the JWT. REQUIRED only if `typ` is `status-assertion-request+jwt`. | {{RFC7519}} Section 4.1.7 |
+| **cti** | Unique identifier for the CWT. REQUIRED only if `typ` is `status-assertion-request+cwt`. | {{RFC7519}} Section 4.1.7 |
+| **credential_hash** | Hash value of the Digital Credential's Issuer signed part the Status Assertion is bound to. | this specification |
+| **credential_hash_alg** | The hash algorithm MUST match the one specified in the `status.status_assertion.credential_hash_alg` claim of the Digital Credential for which the Status Assertion is requested. | this specification |
```

Comment:
```suggestion
| **credential_hash_alg** | The hash algorithm MUST match the one specified in the `status.status_assertion.credential_hash_alg` parameter of the Digital Credential for which the Status Assertion is requested. | this specification |
```

in this case it is a parameter and not a claim from my perspective

Reply:
Claims are assertions about a subject (such as a user) that can be verified. They are often used in security tokens, like JSON Web Tokens (JWT), to convey information about the user.

Parameters in JWTs and metadata help define how the token should be processed or how a service should be accessed.